core/src/main/scala/cats/Inject.scala

```diff
@@ -0,0 +1,54 @@
+package cats
+
+/**
+ * Inject is a type class to inject a value of type `A` into a
+ * coproduct of types `B` when `A :≺: B`. The coproduct `B` is
+ * modelled with `Either`.
```

Comment:
Yes, that's correct. I suppose it could be reworded to indicate that that's just the default behavior?
```
Inject is a type class to inject a value of type `A` into another type `B`.
The default instances provided by cats injects `A` into coproduct of types
`B` when `A :≺: B`. The coproduct `B` is modelled with `Either`.
```

Also in this explanation, `:≺:` is used in a conceptual sense. `:≺:` exists as a type alias for `InjectK` which could be confusing. Thoughts?